### PR TITLE
Add ET Freemius integration

### PR DESCRIPTION
**Pull request by sc0ttkclark**
[ET-595]

## Screencasts

* TEC + ET opt-ins: https://share.getcloudapp.com/E0uEe57W
* ET only: https://share.getcloudapp.com/5zuJKZ5k
* TEC already opted-in and activated ET: https://share.getcloudapp.com/xQu0oKgE

[ET-595]: https://moderntribe.atlassian.net/browse/ET-595

### CODE REVIEW

**Comment by cliffordp** [comment]
@sc0ttkclark i tried it again this morning... full details in [the ticket comment I just left](https://moderntribe.atlassian.net/browse/ET-595?focusedCommentId=49266&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-49266)... but it looks good to go now.

File: src/resources/postcss/freemius.pcss [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+				width: auto;
+			}
+
+			.fs-site-icon {
+				.dashicons {
+					visibility: hidden;
+				}
+
+				&::before {
+					background-image: url( ../images/event-tickets.png );
```
Comment (sc0ttkclark) [inline src/resources/postcss/freemius.pcss]: @cliffordp Let's get the new SVG in place

Comment (sc0ttkclark) [reply src/resources/postcss/freemius.pcss]: We can delete the png that we added as part of this PR

Comment (sc0ttkclark) [reply src/resources/postcss/freemius.pcss]: https://ps.w.org/event-tickets/assets/icon.svg

Comment (cliffordp) [reply src/resources/postcss/freemius.pcss]: done

**Comment by cliffordp** [comment]
Looks good with SVG (but notice the issue with `https://example.com/wp-content/themes/https://example.com/wp-content/plugins/event-tickets/src/resources/images/event-tickets.png`, which existed prior to my commits):

image

**Comment by sc0ttkclark** [comment]
I just confirmed, that's a bug with Freemius itself, no matter the icon value we use for TEC or ET it comes up wrong. I just tested this locally, all looks great now.

**Pull request merged**